domains/itzhenry.json

```diff
@@ -4,8 +4,6 @@
         "email": "trscrus@gmail.com"
     },
     "record": {
-        "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": "itzhenry1.github.io"
```

Comment:
404 CNAME record.

Reply:
Done, put "repo" thing in JSON file like the documentation says.